src/explore-education-statistics-frontend/src/services/sitemapService.ts

```diff
@@ -0,0 +1,60 @@
+'use server';
+
+import { PublicationListSummary } from '@common/services/publicationService';
+import { PaginatedList } from '@common/services/types/pagination';
+import { MetadataRoute } from 'next';
+
+export default async function getPublicationSlugs() {
+  try {
+    // TODO: Replace this with use of the publicationService, once the app router is more widely adopted
+    // (using it currently requires more changes)
+    const publications = (
+      await (
+        await fetch(
+          // TODO: Create a more graceful way of opting out of pagination,
+          // Or make multiple paginated calls, whatever best
+          `${process.env.CONTENT_API_BASE_URL}/publications?pageSize=10000`,
+          {
+            method: 'GET',
+            headers: {
+              'Content-Type': 'application/json',
+            },
+          },
+        )
+      ).json<PaginatedList<PublicationListSummary>>()
+    ).results;
+
+    const routes: MetadataRoute.Sitemap = publications.flatMap(publication => {
+      return [
+        {
+          url: `${process.env.PUBLIC_URL}data-catalogue/${publication.slug}`,
+          lastModified: publication.published ?? new Date(),
+          changeFrequency: 'monthly',
```

Comment:
Updates can be more frequent than monthly. For example, COVID releases were weekly.

Reply:
Agreed - the frequencies are, like the priorities, mostly guesswork so far. I'm particularly dubious about just newing up Dates here too for that matter.